src/test/java/seedu/address/logic/parser/ExportCommandParserTest.java

```diff
@@ -0,0 +1,26 @@
+package seedu.address.logic.parser;
+
+import static seedu.address.commons.core.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+import static seedu.address.logic.parser.CommandParserTestUtil.assertParseFailure;
+import static seedu.address.logic.parser.CommandParserTestUtil.assertParseSuccess;
+
+import org.junit.Test;
+
+import seedu.address.logic.commands.ExportCommand;
+
+public class ExportCommandParserTest {
+
+    private ExportCommandParser parser = new ExportCommandParser();
+
+    @Test
+    public void parse_validArgs_returnsExportCommand() {
+        String args = " some/path/somewhere  ";
```

Comment:
Is the extra spacing intended? 🤔

Reply:
yes, it's to test that the leading/trailing spaces are trimmed!